### PR TITLE
bump to v0.1.5

### PR DESCRIPTION
**Pull request by freeznet**

### CODE REVIEW

**Comment by freeznet** [comment]
@nlu90 @tuteng @Huanli-Meng please help to review this PR if you have time, thanks.

**Review by tuteng — APPROVED** [review]

**Review by nlu90 — APPROVED** [review]

**Comment by nlu90** [comment]
Do we want to wait for some more potential fix PRs before merge this?

**Pull request merged**
